### DISCUSSION TITLE
Add support for arm devices

### DISCUSSION THREAD

**Discussion by Phinixprono123** [kimlimjustin/xplorer#310] [Ideas]
The app is one the best file managers on Linux but the bad side is it doesn't support arm64 device. I saw the screenshots and I knew this is the best file explorer but only for that feature idk what to say anymore.

**Comment by curtisy1** [comment]
It does work on arm64 but you have to build it yourself at the moment. I *think* GitHub now supports arm64 runners in GitHub Actions, so a workflow could be added that also builds and publishes them

**Reply by Phinixprono123** [reply]
Like how can we build it ourself

**Reply by curtisy1** [reply]
Instructions for a development build are available in the [Readme](https://github.com/kimlimjustin/xplorer)

For a release build, you can replace `yarn dev` with `yarn build` and it should work.  Depending on your OS, you might need to also install dependencies.

**Reply by curtisy1** [reply]
![image](https://github.com/user-attachments/assets/20426ff7-25a7-4081-bc7e-09815a7307c5)

Built it on Fedora Asahi Remix. Note that on Fedora, you need to upgrade Tauri to at least 1.4 since otherwise Tauri can't find node. This might also affect other distros but I'm not sure how they handle node shims.
